uint64 sequence
uint64 publish_ts
uint16 stream_id
uint32 width
uint32 height
uint32 channels
uint32 payload_bytes
uint8[4147200] data
